src/routes/event/add.tsx

```diff
@@ -0,0 +1,103 @@
+// WIP
+import { Button, Container, Stack, TextField, Typography } from "@mui/material";
+import { onAuthStateChanged } from "firebase/auth";
+import { useNavigate } from "react-router-dom";
+import { auth } from "../../firebase";
+import AutoComplete, { createFilterOptions } from "@mui/material/Autocomplete";
+import * as React from "react";
+
+const filter = createFilterOptions<string>();
+
+export default function EventAdd() {
+  const navigate = useNavigate();
+  onAuthStateChanged(auth, (user) => {
+    if (!user) {
+      console.log("No user signed in");
+      navigate(-1);
```

Comment:
This will not work if we go on this page directly and we have no history.
Its better to redirect to a specific page instead if going "Back"